lib/brcobranca/boleto/santander_com_repasse_de_regras.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Brcobranca
+  module Boleto
+    class SantanderComRepasseDeRegras < BaseSemOverride
+      attr_accessor :seu_numero, :codigo_cedente, :codigo_barras
+      attr_reader :linha_digitavel, :nosso_numero_boleto
+
+      validates_length_of :agencia, maximum: 4, message: 'deve ser menor ou igual a 4 dígitos.'
+      validates_length_of :convenio, maximum: 9, message: 'deve ser menor ou igual a 9 dígitos.'
+      validates_length_of :numero_documento, maximum: 12,
```

Comment:
Se o comentário acima não for acatado deve ser removida essa linha certo?

Reply:
Tens razão, gajo :portugal:

Reply:
Done!